app/views/users/join.ejs

```diff
@@ -15,7 +15,9 @@ this license in a file with the distribution.
 	</header>
 	<!-- current text on the join page for liquid  -->
 	<p>
-		#TODO: @tommypacker fill this in/give us the text to fill this in.
+		Joining ACM is completely free! Once you've found your niche in ACM or one of its SIGs, becoming an official member is easy. To become an official member, you need to fill out the quick and easy form below.
+
+If you would like to support ACM’s operations, you can pay a onetime $40 fee that grants you access to our many perks including free printing, Merch credit, and cluster use. You can pay online here: (insert url), or come into our office between 9:00 AM and 5:00 PM to pay in cash/Venmo.
 	</p>
 
```

Comment:
@bcongdon @sameetandpotatoes something for a new dev to take on. After the join page is submitted we need to figure out post sign up behavior. (do we show stripe, how is this found if someone has already filled the form but did it earlier, we also want to show the groups to join as well)